lib/ddtrace/contrib/httparty/request_patch.rb

```diff
@@ -0,0 +1,137 @@
+require 'ddtrace/ext/net'
+require 'ddtrace/ext/distributed'
+require 'ddtrace/propagation/http_propagator'
+require 'ddtrace/contrib/rest_client/ext'
+
+module Datadog
+  module Contrib
+    module HTTParty
+      # HTTParty RequestPatch
+      module RequestPatch
```

Comment:
Also see my other suggestion above, which would have this change to something like `Datadog::Contrib::HTTParty::Instrumentation::Request`.